.github/workflows/build_and_release.yml

```diff
@@ -470,31 +470,44 @@ jobs:
 
 
       - name: Setup our dev environment
-        uses: ilammy/msvc-dev-cmd@v1.4.1
+        uses: ilammy/msvc-dev-cmd@v1
 
       - name: Setup ninja cmake generator
         uses: seanmiddleditch/gha-setup-ninja@master
 
       - name: Download and build our dependencies using vcpkg
-        uses: lukka/run-vcpkg@v10.2
+        uses: Lord-Kamina/vcpkg-action@master
         with:
-          vcpkgGitCommitId: '14e7bb4ae24616ec54ff6b2f6ef4e8659434ea44'
-          runVcpkgInstall: true
+          pkgs: aubio boost-iostreams boost-locale boost-program-options boost-system cairo cpprestsdk ffmpeg fftw3 fmt fontconfig gdk-pixbuf gettext glibmm glm gtest icu libepoxy librsvg libsigcpp libxml2 libxmlpp nlohmann-json openblas opencv openssl pango portaudio portmidi pthreads sdl2
+          triplet: x64-windows-release
+          extra-args: --clean-after-build
+          cache-key: win64-vcpkg
 
       - name: Run cmake to configure the project and build it
         env:
           PERFORMOUS_VERSION: ${{ needs.determine_version.outputs.complete_version }}
         run: |
+          set VCPKG_DEFAULT_BINARY_CACHE=${{ github.workspace }}/vcpkg_cache
           cd ${{ github.workspace }}/performous
           echo $PERFORMOUS_VERSION
-          cmake --preset "x64-release"
-          cmake --build --preset "x64-release"
-
+          cmake --preset "x64-release" -DCMAKE_TOOLCHAIN_FILE=${{ github.workspace }}/vcpkg/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=x64-windows-release
+          cmake --build --preset "x64-release" -DVCPKG_TARGET_TRIPLET=x64-windows-release -DCMAKE_TOOLCHAIN_FILE=${{ github.workspace }}/vcpkg/scripts/buildsystems/vcpkg.cmake
+      - name: Run unittests directly called.
+        shell: powershell
+        run: |
+          cd "performous/build/x64-release/testing"
+          ./performous_test.exe --gtest_filter=UnitTest*
+      - name: Run unittests by ctest.
+        shell: powershell
+        run: |
+          cd "performous/build/x64-release"
+          ninja testing/test
       - name: Create Installer
         id: package_performous_x64
+        shell: powershell
```

Comment:
the shell is per definition powershell on windows machines so no need to specify this

Reply:
Pretty sure else the script below wouldn't work aswell.
Maybe instead of setting it for each step, define it globally within this pipeline. The GH-action you're using works with either linux, windows or osx